### ISSUE TITLE
Button does not have an accessible name

### ISSUE THREAD

**Issue by alexdor** [ossn/fixme#25]
When a button doesn't have an accessible name, screen readers announce it as "button", making it unusable for users who rely on screen readers.

Failing elements
```
<button type="button" class="navbar-toggler"><span class="navbar-toggler-icon"></span></button>
```
